worker/src/RTC/Router.cpp

```diff
@@ -780,24 +780,31 @@ namespace RTC
 	{
 		MS_TRACE();
 
-		auto& consumers = this->mapProducerConsumers.at(producer);
+		std::shared_ptr<RTC::RtpPacket> sharedPacket{ nullptr };
 
-		if (!consumers.empty())
+		// The packet does not need to be accessible in the future, do not clone it.
+		if (producer->GetKind() == RTC::Media::Kind::AUDIO)
+		{
+			std::shared_ptr<RTC::RtpPacket> sharedPacket(packet);
```

Comment:
I don't think it does what you think it does, this is probably more correct:
```suggestion
			sharedPacket.reset(packet);
```

But it shouldn't compile because it is just a pointer and can't be de-allocated.

Reply:
Yep, it's been fixed in the last commit.